src/Payum/Paypal/ExpressCheckout/Nvp/Api.php

```diff
@@ -426,6 +426,24 @@ public function doExpressCheckoutPayment(array $fields)
      *
      * @return array
      */
+    public function cancelRecurringPaymentsProfile(array $fields)
+    {
+        $request = new FormRequest();
+        $request->setFields($fields);
+
+        $request->setField('METHOD', 'CancelRecurringPaymentsProfile');
```

Comment:
cannot find this `CancelRecurringPaymentsProfile` Paypal method. Could you point me to the doc